runtime/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use rstd::{vec::Vec, collections::btree_map::BTreeMap};
+use codec::{Encode, Decode};
+use runtime_primitives::traits::Block as BlockT;
+
+#[cfg(not(feature = "std"))]
+#[doc(hidden)]
+pub use rstd::slice;
+
+#[macro_use]
+pub mod validate_block;
+
+type WitnessData = BTreeMap<Vec<u8>, Vec<u8>>;
+
+/// The parachain block that is created on a collator and validated by a validator.
+#[derive(Encode, Decode)]
+struct ParachainBlock<B: BlockT> {
```

Comment:
Let's call this `ParachainBlockData` to be consistent with Polkadot terminology.
The doc would be "block data needed to validate a state transition for a Cumulus parachain. Created by collators, executed by validators". Or something.